cmd/suggestion/hyperband/v1beta1/Dockerfile

```diff
@@ -5,9 +5,12 @@ ENV TARGET_DIR /opt/katib
 ENV SUGGESTION_DIR cmd/suggestion/hyperband/v1beta1
 ENV PYTHONPATH ${TARGET_DIR}:${TARGET_DIR}/pkg/apis/manager/v1beta1/python:${TARGET_DIR}/pkg/apis/manager/health/python
 
-RUN if [ "${TARGETARCH}" = "ppc64le" ] || [ "${TARGETARCH}" = "arm64" ]; then \
```

Comment:
Any reasons to remove `arm64` from here?

Reply:
No such reason but I think its typo and we can fix it by nested if-else condition.

Reply:
Sorry, I couldn't follow it. where is a typo?
Could you clarify it? Thanks.

Reply:
I think, these packages are required to support arm64, isn't ?

Reply:
We verify grpcio installation only for ppc64le  but not for arm64 and while raising the PR, I think arm64 has been mistakenly removed in hyperband dockerfile. So will update dockerfile and add a condition where if it is ppc64le then only install grpcio from source...

Reply:
> We verify grpcio installation only for ppc64le but not for arm64 and while raising the PR, I think arm64 has been mistakenly removed in hyperband dockerfile. So will update dockerfile and add a condition where if it is ppc64le then only install grpcio from source...

Ah, I see. Thanks.